webapp/src/domain/tests/AllTests.tsx

```diff
@@ -282,6 +286,27 @@ export default function AllTests() {
             return <WatchDropdown watching={arg.cell.value} id={arg.row.original.id} />
         },
     }
+     
+    const [page, setPage] = useState(1)
+    const [perPage, setPerPage] = useState(20)
+    const [direction] = useState<SortDirection>("Ascending")
+    const pagination = useMemo(() => ({ page, perPage, direction }), [page, perPage, direction])
+    const [tests, setTets] = useState<TestQueryResult>()
+    const [loading, setLoading] = useState(false)
+
+    useEffect(() => {
+        setLoading(true)
+        Api.testServiceList(
+             SortDirection.Ascending,
+             pagination.perPage,
+             pagination.page
```

Comment:
this causes an off-by-one error as the REST API is zero-based and the JS library is one-based,

see: https://github.com/Hyperfoil/Horreum/pull/532/files#diff-4529692787927fec91b2082b9832b678a10586615349fa8663cc0026dcd63c65R38

Reply:
Thanks!